engines/mohawk/myst.cpp

```diff
@@ -942,6 +967,21 @@ void MohawkEngine_Myst::changeToCard(uint16 card, TransitionType transition) {
 		_card->leave();
 	}
 
+	// Add artificial CD-ROM delay
+	if (addCdRomDelay == true) {
+		if (_stack->getStackId() != kIntroStack && _stack->getStackId() != kMenuStack) {
+			// The original engine disables the mouse cursor when loading new cards.
```

Comment:
I had an impression that it shows a CD-ROM icon.  Like seen here, for example: https://youtu.be/9F9ahZQ7oP0?t=2793

Reply:
Yes, you are right. However, the CD-ROM icon was a version exclusive for the PS1 and (IIRC) the Amiga CD32 version. The Windows/Mac versions I had available for testing just blank the cursor until the next card is shown.